packages/uploadthing/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uploadthing",
-  "version": "5.7.1",
+  "version": "6.7.1",
```

Comment:
REVERT

Reply:
What is this and why did it happen?

Reply:
it's what i use for testing to get the correct handler on the infra side as that's decided by the sdk version :)